app/models/audio_file_template.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+class AudioFileTemplate < BaseModel
+  belongs_to :audio_version_template
+  acts_as_list scope: :audio_version_template
+
+  before_validation :set_defaults, on: :create
+
+  validates :length_minimum,
+            presence: true,
+            numericality: { less_than_or_equal_to: :length_maximum }
+
+  validates :length_maximum,
+            presence: true,
+            numericality: { greater_than_or_equal_to: :length_minimum }
+
+  def set_defaults
+    self.label ||= 'segment'
```

Comment:
Nah - just making sure I understood that.  For Publish, label will probably be a required field anyways.  So they'll have to create a non-default one.